core/src/main.rs

```diff
@@ -320,8 +327,9 @@ fn handle_oracle_command(command: Command) {
 }
 
 fn main_loop_iteration(op: &OraclePool, read_only: bool) -> std::result::Result<(), anyhow::Error> {
-    let height = current_block_height()?;
+    let height = current_block_height()? as u32;
     let wallet = WalletData::new();
+    let networt_change_address = get_change_address_from_node()?;
```

Comment:
Just a small typo. It should read `network_change_address`.

Reply:
Thank you! Fixed.